massa-storage/src/lib.rs

```diff
@@ -113,7 +152,11 @@ impl Storage {
             "operation_id": operation_id
         });
         let operations = self.operations.read();
-        f(&operations.get(operation_id))
+        let (prefix, suffix) = operation_id.split();
+        f(&match operations.get(&prefix) {
```

Comment:
use `operations.get(&prefix).map(|ops| ops.get(&prefix))`?

Reply:
Good :+1: I'll use `operations.get(&prefix).and_then(|ops| ops.get(&prefix))` to `flatten` the nested options